src/pages/scheduler/index.tsx

```diff
@@ -2,16 +2,34 @@ import { Flex } from '@chakra-ui/layout';
 import { useParams } from 'react-router';
```

Comment:
lint: `useParams` is defined but never used

Reply:
This is from putting in event so it can be seen on a calendar. It will be reverted before I merge. Same for other comment